src/VsixTesting.Installer/Program.cs

```diff
@@ -649,4 +665,116 @@ internal static class AppDomainExtensions
         public static T CreateInstanceFromAndUnwrap<T>(this AppDomain domain)
             => (T)domain.CreateInstanceFromAndUnwrap(typeof(T).Assembly.Location, typeof(T).FullName);
     }
+
+    internal class VisualStudioUtil
```

Comment:
❓ Can we not use a linked file so one implementation file works in both projects that need it? Applies to each of the helper classes added here.

Reply:
Yes, we can but my evil plan of considering moving the installer to its own repo wouldn't be possible anymore.

I don't know if its worth it. I wonder what you think about it.

Should we have all the projects in a single repo, or perhaps split them:
* https://github.com/josetr/VsixTesting
* https://github.com/josetr/VsixTesting.Installer
* https://github.com/josetr/VsixTesting.VisualStudio/tree/dev (I was going to merge this repo back to VsixTesting but I no longer know if I should. This is a VS extension containing a project template which creates a "VSIX Test Project" for xunit but we can add anything useful here)

My fear is that CI builds will be perhaps too slow when we switch to azure pipelines once they install VS2012/VS2013/VS2015 on their VS2017 image. I'm almost sold on their 10 concurrent builds :smile:.

Reply:
Either way works. If it's split, you could provide the implementation of these types in exclusively VsixTesting.Installer, and reference that tool from VsixTesting as a library rather than duplicating it. In the end whatever is easiest for you is fine.

It would be nice to have some of those additional images available for builds. I'm not sure if/when that would happen though.

Reply:
I'd be inclined to keep them as a single repository unless there is a compelling reason to split them. Proliferating repositories and submodules tend to cause maintenance hassles.